src/gwt/test/org/rstudio/core/client/StringUtilTests.java

```diff
@@ -238,6 +238,40 @@ public void testConciseElapsedTime()
       assertEquals("2:02:01:01", days);
    }
   
+   public void testGetCssIdentifier()
+   {
+      for (int pass = 0; pass < 5; pass++)
```

Comment:
Nit: I think this might be frustrating to update since you have to remember to increase the pass count here whenever you add a pass below. How about just putting the input/expected values in arrays, and looping over the arrays?

Reply:
Updated!